app/presenters/activity_csv_presenter.rb

```diff
@@ -7,4 +7,9 @@ def intended_beneficiaries
   def beis_id
     super.to_s
   end
+
+  def country_delivery_partners
+    return if super.blank?
+    to_model.country_delivery_partners.join(" | ")
```

Comment:
Correct me if I misunderstand it, but I thought we only need to call `to_model` if the activity presenter has a different implementation than what we need in the CSV presenter. In this case, it looks like the activity presenter doesn't process this field in any way, so could we use `super` like usual? (I might have things wrong, considering I didn't know about `to_model` until now. :) )

Reply:
Not sure. I'll have a look